modules/steam/hm.nix

```diff
@@ -0,0 +1,39 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.stylix.targets.steam;
+in {
+  options.stylix.targets.steam = {
+    enable = config.lib.stylix.mkEnableTarget "Steam" true;
+    adwaitaForSteam.enable = config.lib.stylix.mkEnableTarget "Adwaita for Steam" false;
+  };
+
+  config = lib.mkIf (config.stylix.enable && cfg.enable) (lib.mkMerge [
+    {
+      # Generic configuration
+    }
+
+    (lib.mkIf cfg.adwaitaForSteam.enable {
+      home.packages = with pkgs; [adwsteamgtk];
+
+      home.activation.adwaitaForSteam = let
```

Comment:
> In case the uncontained nature of the activation script is a problem, I can try to implement the solution suggested in https://github.com/danth/stylix/issues/551#issuecomment-2378167369.

Is it possible to make this reproducible? `${lib.getExe pkgs.adwsteamgtk} --install` does not sound very reproducible.

Reply:
One could probably run this script in a stdenv and extract the relevant files, or betted yet, extract them from the source repository.

Reply:
Extracting the necessary files from the source repo is definitely the best solution, as the activation hook hack isn't easily reversible should a user choose to disable it

Reply:
Agreed.